var foo = function() {
    $throw("failure");
}
try {
	foo()
} catch e {
      $print(e); // prints "failure"
}
